julia/src/compile.jl

```diff
@@ -17,23 +17,31 @@ end
 
 
 """
-    get_bridgestan_path() -> String
+    get_bridgestan_path(;download=true) -> String
 
 Return the path the the BridgeStan directory.
 
-If the environment variable `BRIDGESTAN` is set, this will be returned.
-Otherwise, this function downloads a matching version of BridgeStan under
-a folder called `.bridgestan` in the user's home directory.
+If the environment variable `\$BRIDGESTAN` is set, this will be returned.
+
+If `\$BRIDGESTAN` is not set and `download` is true, this function downloads
+a matching version of BridgeStan under a folder called `.bridgestan` in the
```

Comment:
I've forgotten about what matching means here, so maybe we could make it more explicit?